build.gradle.kts

```diff
@@ -47,6 +47,13 @@ tasks.shadowJar {
     archiveClassifier = ""
 }
 
+tasks.withType<Jar> {
```

Comment:
Do you need to use `configure.each` to avoid early task instantiation?
https://docs.gradle.org/current/userguide/lazy_configuration.html#from_a_build_script

Reply:
You are correct. Fixed in most recent changes. 82204a7e67fe67face714da6d453e63ed8f68e3d